src/test/java/org/folio/rest/impl/AutomatedPatronBlocksAPITest.java

```diff
@@ -143,8 +143,14 @@ public void shouldReturnNoBlocksWhenUserSummaryDoesNotExist() {
 
   @Test
   public void shouldReturnNoBlocksWhenNoLimitsExistForPatronGroup() {
+    final String userId = randomId();
+    UserSummary userSummary = new UserSummary()
+      .withId(randomId())
+      .withUserId(userId);
+    waitFor(summaryRepository.save(userSummary));
+
```

Comment:
Why should we add this?

Reply:
There is one check before reaching the point where the test can work, so before without adding user summary the work flow even earlier breaks and returns without reaching the point.

`public Future<AutomatedPatronBlocks> getBlocksForUser(String userId) {
    return userSummaryService.getByUserId(userId)
      .map(userSummary -> new BlocksCalculationContext().withUserSummary(userSummary))
      .compose(this::getBlocksForSummary)
      .otherwise(new AutomatedPatronBlocks());
  }
`
here firstly it gets userSummery by userId, the proceeds to getBlocksForSummary, before the test was written without this, then my code change was not covered, since it returns here by failing to get user summary.

`public Future<UserSummary> getByUserId(String userId) {
    return userSummaryRepository.getByUserId(userId)
      .map(optionalUserSummary -> optionalUserSummary.orElseThrow(() ->
        new EntityNotFoundInDbException(format("User summary for user ID %s not found", userId))));
  }`
here it would return with exception if I don't define user summary in here (test)